recipes/pyutilsnrw/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyutilsnrw" %}
+{% set version = "0.1.0" %}
+{% set hash_type = "md5" %}
```

Comment:
Please use the `sha256` hash. Grabbing it from pypi.io:

`79db121ab7b69c526c8a13769d16e2311e952092ed09b828099bbfb755a16c51`